### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ryanwohara**
:gear: **Issue**

Clarify that the API token to be used should be the privileged one from the k8s secret

:white_check_mark: **Fix**

:question: **Tests**

- [ ] Tested Updating Existing Instance
- [ ] Installed on new instance

### CODE REVIEW

**Review by atulsingh0 — COMMENTED** [review]
Are we talking abt `CIRCLECI_TOKEN` variable here?

File: vault-to-tink/README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 1. The kubectl context needs to be in the correct namespace (`kubectl config set-context --current --namespace <namespace>`)
 1. Docker is running and accessible without `sudo` to run the `clojure` container
 1. `jq` is installed.
+1. The API token from your Kubernetes secret is accessible to be used for the import.
```
Comment (atulsingh0) [inline vault-to-tink/README.md:9]: are we taking abt `CIRCLECI_TOKEN` here?

Comment (ryanwohara) [reply vault-to-tink/README.md:9]: It's `api-token` or `CIRCLE_INTERNAL_API_TOKEN`  or `CIRCLE_BOT_TOKEN` - it has a lot of names in our chart 😅

Comment (atulsingh0) [reply vault-to-tink/README.md:9]: and that is why, let's add this as well. `token listed here - CIRCLE_HOSTNAME/account/api`

Comment (atulsingh0) [reply vault-to-tink/README.md:9]: so let's add this as well - `kubectl get secret api-token -o yaml | yq '.data.api-token' | base64 -d`

Comment (atulsingh0) [reply vault-to-tink/README.md:9]: or may be add in the script itself rather than asking user.

Comment (ryanwohara) [reply vault-to-tink/README.md:9]: I do want to add it to the script but I figured it wasn't worth the rush to get it in before we release 4.2
I'll file a ticket for it

**Review by atulsingh0 — APPROVED** [review]

File: vault-to-tink/README.md (hunk 1 above)
Comment (ryanwohara) [reply vault-to-tink/README.md:9]: https://circleci.atlassian.net/browse/ONPREM-29

**Pull request merged**